fix demo parsing

panic: interface conversion: interface {} is uint64, not bool [recovered]
        panic: interface conversion: interface {} is uint64, not bool

goroutine 7 [running]:
testing.tRunner.func1.2({0xea48a0, 0xc0011fd290})
        C:/Program Files/Go/src/testing/testing.go:1734 +0x219
testing.tRunner.func1()
panic({0xea48a0?, 0xc0011fd290?})
panic({0xea48a0?, 0xc0011fd290?})
        C:/Program Files/Go/src/runtime/panic.go:792 +0x132
github.com/markus-wa/demoinfocs-golang/v5/pkg/demoinfocs/sendtables.PropertyValue.BoolVal(...)
        C:/Users/michal/go/pkg/mod/github.com/markus-wa/demoinfocs-golang/v5@v5.0.2/pkg/demoinfocs/sendtables/sendtables.go:94 
csgo-2d-demo-player/pkg/parser.createTickStateMessage({0x1034280, 0xc000134180}, 0xc0003cc330, {0x1033ba0, 0xc000314000}, 0xc0003cce70)
        C:/Users/michal/dev/csgo-2d-demo-viewer/pkg/parser/parser.go:270 +0xdba
csgo-2d-demo-player/pkg/parser.parseMatch({0x1033ba0, 0xc000314000}, 0xc000052e90)
        C:/Users/michal/dev/csgo-2d-demo-viewer/pkg/parser/parser.go:210 +0xcb5
csgo-2d-demo-player/pkg/parser.Parse({0x1027c20?, 0xc000278400?}, 0xc000052e90)
        C:/Users/michal/dev/csgo-2d-demo-viewer/pkg/parser/parser.go:35 +0xb3
csgo-2d-demo-player/pkg/parser.WasmParseDemo({0xf9eccf?, 0x42?}, {0x1027b80?, 0xc000092228?}, 0xfbe808)
        C:/Users/michal/dev/csgo-2d-demo-viewer/pkg/parser/wasmparser.go:21 +0x85
csgo-2d-demo-player/pkg/parser.TestParseZstDemoArchive(0xc000003a40)
        C:/Users/michal/dev/csgo-2d-demo-viewer/pkg/parser/wasmparser_test.go:17 +0xb6
testing.tRunner(0xc000003a40, 0xfbe718)
        C:/Program Files/Go/src/testing/testing.go:1792 +0xcb
created by testing.(*T).Run in goroutine 1
        C:/Program Files/Go/src/testing/testing.go:1851 +0x3f6
FAIL    csgo-2d-demo-player/pkg/parser  4.818s
FAIL

--- a/pkg/parser/parser.go
+++ b/pkg/parser/parser.go
@@ -267,7 +267,7 @@ func createTickStateMessage(tick dem.GameState, mapCS *MapCS, parser dem.Parser,
 		var action string
 		if g.WeaponInstance.Type == common.EqHE {
 			// HE for some reason keep on map longer. we want to remove them after they explode
-			if exploded, ok := g.Entity.PropertyValue("m_nExplodeEffectIndex"); ok && exploded.BoolVal() {
+			if exploded, ok := g.Entity.PropertyValue("m_nExplodeEffectIndex"); ok && exploded.UInt64() > 0 {
 				continue
 			}
 		}

